test/Test/Fixtures.hs

```diff
@@ -206,3 +217,56 @@ issuesPending =
 
 attributionReportAsSerializedJson :: Text
 attributionReportAsSerializedJson = "{\"TestReport\": \"TestReportData\"}"
+
+signedUrl :: API.SignedURL
+signedUrl = API.SignedURL{API.signedURL = "https://foo.com"}
+
+-- Test data for licenseScanSourceUnits tests
+firstVendoredDep :: VendoredDependency
+firstVendoredDep =
+  VendoredDependency
+    "first-archive-test"
+    "vendor/foo"
+    (Just "0.0.1")
+
+secondVendoredDep :: VendoredDependency
+secondVendoredDep =
+  VendoredDependency
+    "second-archive-test"
+    "vendor/bar"
+    (Just "0.0.1")
+
+vendoredDeps :: NonEmpty VendoredDependency
+vendoredDeps = firstVendoredDep :| [secondVendoredDep]
```

Comment:
I'll get rid of these `:|` in a future PR by changing out hlint config to allow `NE.fromList` in this file and in specs.

Reply:
Because `:|` makes me feel :|

Reply:
It really doesn't seem necessary for fixtures when you passing a list that is defined in-line.  I'd support that!